vnext/Microsoft.ReactNative.IntegrationTests/ReactNativeHostTests.cpp

```diff
@@ -1,4 +1,5 @@
 #include "pch.h"
+#include "NativeModules.h"
```

Comment:
I think I have implemented this change too. It must have been submitted already.